CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- Replace switchable controller actions with separate plugins (#569)
```

Comment:
We're usually having the PR ID here, not the issue ID. So this should be #575 instead.

Reply:
Ok